Sources/ConsoleKit/Terminal/readpassphrase_linux.swift

```diff
@@ -15,7 +15,7 @@ import Dispatch
 /// fatal error to call it off the main thread. This enables us to have the signal recovery handler write to somewhere
 /// it can find without risking intervention from the Swift runtime at async-signal time.
 internal func linux_readpassphrase(_ prompt: UnsafePointer<Int8>, _ buf: UnsafeMutablePointer<Int8>, _ bufsiz: Int, _ flags: Int32) -> UnsafeMutablePointer<Int8>? {
-    dispatchPrecondition(condition: .onQueue(.main))
+    //dispatchPrecondition(condition: .onQueue(.main))
```

Comment:
If we're not using this check anymore we should just remove it instead of commenting it out

Reply:
It's not so much that we're not using it as that we can't at the moment enforce it properly.